.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,39 +1,34 @@
+<!-- prettier-ignore -->
 ## Fixes
-
 <!-- If PR doesn't fully resolve the issue, replace 'Fixes' below with 'Related to'. -->
 <!-- If there is no issue being resolved, please consider opening one before creating this pull request. -->
 
 Fixes #[issue number] by @[issue author]
 
 ## Description
-
 <!-- Concisely describe what the pull request does. -->
 <!-- Add screenshots, videos, or other media to show the problem and the solution when appropriate. -->
 
-## Testing Instructions
 
+## Testing Instructions
 <!-- Give steps for the reviewer to verify that this PR fixes the problem; or delete this section entirely. -->
 
-## Checklist
 
+## Checklist
 <!-- Replace  the [ ] with [x] to check the boxes. -->
 
-- [ ] My pull request has a descriptive title (not a vague title like
-      `Update index.md`).
-- [ ] My pull request targets the _default_ branch of the repository (`main`) or
-      a parent feature branch.
+- [ ] My pull request has a descriptive title (not a vague title like`Update index.md`).
+- [ ] My pull request targets the _default_ branch of the repository (`main`) or a parent feature branch.
 - [ ] My commit messages follow [best practices][best_practices].
 - [ ] My code follows the established code style of the repository.
 - [ ] I added or updated tests for the changes I made (if applicable).
 - [ ] I added or updated documentation (if applicable).
-- [ ] I tried running the project locally and verified that there are no visible
-      errors.
+- [ ] I tried running the project locally and verified that there are no visible errors.
```

Comment:
Oh while we're here, do you think it makes sense to add back in this checklist item from the catalog repo now that it's part of this project as well?
```suggestion
- [ ] I tried running the project locally and verified that there are no visible errors.
- [ ] I ran the DAG documentation generator (if applicable).
```